drivers/sx126x/sx126x_netdev.c

```diff
@@ -288,8 +288,9 @@ static int _set_state(sx126x_t *dev, netopt_state_t state)
     case NETOPT_STATE_RX:
         DEBUG("[sx126x] netdev: set NETOPT_STATE_RX state\n");
         sx126x_cfg_rx_boosted(dev, true);
-        if (dev->rx_timeout != 0) {
-            sx126x_set_rx(dev, dev->rx_timeout);
+        const int _timeout = (sx126x_symbol_to_msec(dev, dev->rx_timeout));
```

Comment:
Is this really const?

Reply:
Just to be safe, compile optimisation and adding some clarity. This data should only be set via `sx126x_symbol_to_msec` , wrong values will break the stack.

Reply:
const have no meaning here. It should be dropped.

Reply:
Oops. Thanks, Jose for the clarification.